### ISSUE TITLE
HTML response is not considered an error? 

### ISSUE THREAD

**Issue by pke** [mzabriskie/axios#717]
Without setting an `Accept` header responses in HTML of content type `text/html` are not considered an error by axios by default?
It seems to wrap the response, although being 200 - OK into a `{data: <html>blabla</html>, status: 200}`. How can I enforce json responses and everything else considered an error?

**Comment by KaelWD** [comment]
Use a response interceptor (assuming your server is sending the correct content-type):
```js
axios.interceptors.response.use(response => {
  return response.headers['content-type'] === 'application/json' ? response : Promise.reject(response);
}, error => Promise.reject(error));
```

**Comment by rubennorte** [comment]
This is something that will be fixed once #61 is done.

**Comment by Nitiquita** [comment]
I am having the same issue. Can someone tell me where the response interceptor should be placed? Thanks!
